app/controllers/answers_controller.rb

```diff
@@ -1,2 +1,29 @@
 class AnswersController < ApplicationController
+
+  def new
+    @answer = Answer.new
+  end
+
+  def create
+    answer = Answer.new(answer_params)
+    if answer.save
+      redirect_to :back
+    else
+      redirect_to new_answer_path
+  end
```

Comment:
It's common to use render in the else case to render the new form. See how scaffold does this.